cds-snc/covid-alert-portal
# Change the flow/step order and add step counter

# Summary | Résumé

Follow up to (and branched from) #462 

Changes the order of the steps and adds a Step counter to the top of pages (ie, Step 1 of 5...)
- Change order of steps
- Add Step counter at top
- Update back links for reordered steps
- Update translation files

## To test
- Follow the process, new order should be:
    - Registrant Email
    - Email confirm
    - Category (1 of 5)
    - Name of event/establishment (2 of 5)
    - Address (3 of 5)
    - Contact (4 of 5)
    - Check your answers (5 of 5)
